packages/dds/sequence/api-report/sequence.api.md

```diff
@@ -538,8 +539,8 @@ export class SharedIntervalCollectionFactory implements IChannelFactory {
 export abstract class SharedSegmentSequence<T extends ISegment> extends SharedObject<ISharedSegmentSequenceEvents> implements ISharedIntervalCollection<SequenceInterval>, MergeTreeRevertibleDriver {
     constructor(dataStoreRuntime: IFluidDataStoreRuntime, id: string, attributes: IChannelAttributes, segmentFromSpec: (spec: IJSONSegment) => ISegment);
     annotateRange(start: number, end: number, props: PropertySet, combiningOp?: ICombiningOp): void;
-    // (undocumented)
-    protected applyStashedOp(content: any): unknown;
+    // @internal (undocumented)
```

Comment:
I think I added this because of the compiler error that came up when I changed the return type from unknown to the actual types - IMapMessageLocalMetadata is tagged internal, but applyStashedOp is tagged alpha.